### ISSUE TITLE
Failed

### ISSUE THREAD

**Issue by bmsjr** [MicrosoftDocs/mslearn-migrate-sql-server-relational-data#7]
https://docs.microsoft.com/en-us/learn/modules/migrate-sql-server-relational-data/2-exercise-environment-setup - I am getting a failure when provisioning:

![image](https://user-images.githubusercontent.com/3418682/86375962-58202d80-bc54-11ea-8d11-a1f59274c6be.png)

**Comment by Xriztoppa** [comment]
I asked a question about this in the [MS Q&A](https://docs.microsoft.com/en-us/answers/questions/97354/conflict-during-deployment-customscriptextension-r.html). A user has identified the problem:

> It seems something is wrong with this line `Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force | Out-Null;` No Nuget Package provider is found that mets the criteria of the MinimumVersion (my guess)

The script in question is [this one](https://raw.githubusercontent.com/MicrosoftDocs/mslearn-migrate-sql-server-relational-data/master/AddDatabaseToSqlServer.ps1).

@jboeshart could you pls fix this?

**Comment by pauldj54** [comment]
Hi everyone, 
I am facing the same issue.
I would try to repair it but an official fix would be nice.

By deploying the custom script extension I got the error:
```
{
    "status": "Failed",
    "error": {
        "code": "ResourceDeploymentFailure",
        "message": "The resource operation completed with terminal provisioning state 'Failed'.",
        "details": [
            {
                "code": "VMExtensionProvisioningTimeout",
                "message": "Provisioning of VM extension CustomScriptExtension has timed out. Extension provisioning has taken too long to complete. The extension did not report a message. More information on troubleshooting is available at https://aka.ms/VMExtensionCSEWindowsTroubleshoot"
            }
        ]
    }
}
```

**Comment by pauldj54** [comment]
Hi everyone , 

I tried to run the ps script inside the created machine in order to fix the error. I had troubles installing the NuGet package manager and solved the problems as explained here: [Unable to install NuGet provider for PowerShell](https://www.alitajran.com/unable-to-install-nuget-provider-for-powershell/)

It seems to be a problem with the transport layer security protocols. TLS 1.2 must be enable. The minor version is not the issue, I was able to find it using the Find-Package command in PS.

Then I modified the "AddDatabaseToSqlServer.ps1" script accordingly and tried to deploy again. Unfortunately I got a time-out again: [AddDatabaseToSqlServer.ps1](https://github.com/pauldj54/mslearn-migrate-sql-server-relational-data/blob/master/AddDatabaseToSqlServer.ps1)

Finally and in order to be able to finish the exercise I ran the ps script inside the VM opening PS as admin and populating the variables $userName and $password with the correspondent values. 

Exercise done but template still buggy.

Best regards, 
Paul
